src/SDK/SmartDetectorsSDK/ResourceIdentifier.cs

```diff
@@ -43,6 +43,19 @@ public struct ResourceIdentifier : IEquatable<ResourceIdentifier>
         private const string ResourceGroupRegexPattern = SubscriptionRegexPattern + "/resourceGroups/(?<resourceGroupName>[^/]*)";
         private const string ResourceRegexPattern = ResourceGroupRegexPattern + "/providers/(?<resourceProviderAndType>.*)/(?<resourceName>[^/]*)";
 
+        /// <summary>
+        /// A dictionary, mapping <see cref="ServiceType"/> enumeration values to matching presentation in URI
+        /// </summary>
+        private static readonly ReadOnlyDictionary<ServiceType, string> MapAzureServiceTypeToPresentationInUri =
```

Comment:
>PresentationInUri [](start = 93, length = 17)

ResourceUriPath? 
not sure presentation has anything to do with it

Reply:
or follow the pattern of MapResourceTypeToString and name it MapServiceTypeToString

---
In reply to: [281468910](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/63#discussion_r281468910) [](ancestors = 281468910)